samples/app-link-unfurling-in-share-to-teams/csharp/README.md

```diff
@@ -0,0 +1,119 @@
+---
+page_type: sample
+description: This sample demos the feature of link unfurling for Share to teams.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "14-12-2022 00:15:13"
```

Comment:
Metadata for sample.json is missing here.

Reply:
updated